ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,6 +1202,9 @@ literals:
 - leaving_water_tank_temperature_sensor
 - entering_water_tank_temperature_sensor
 
+# Tank level
+- level_percentage_sensor
```

Comment:
or better yet, percentage_sensor is probably fine here because you are applying it to a tank, so what its measuring is obviously the tank. otherwise, might even consider a new subfield analogous to level defined as a measurement descriptor.